cdap-ui/app/cdap/components/CaskWizards/MicroserviceUpload/EndpointStep/index.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import React from 'react';
+import {connect, Provider} from 'react-redux';
+import {Col, FormGroup, Label, Form} from 'reactstrap';
+import InputWithValidations from 'components/InputWithValidations';
+import T from 'i18n-react';
+
+import MicroserviceUploadStore from 'services/WizardStores/MicroserviceUpload/MicroserviceUploadStore';
+import MicroserviceUploadActions from 'services/WizardStores/MicroserviceUpload/MicroserviceUploadActions';
+import DSVInboundQueues from './DSVInboundQueues.js';
```

Comment:
These should be `components/CaskWizards/MicroserviceUpload/EndpoiuntStep` right?